### DISCUSSION TITLE
https://blog.3place.cn/3place/lmyn/LMYN009.html

### DISCUSSION THREAD

**Discussion by giscus** [hbaolong/hbaolong.github.io#55] [General]
# https://blog.3place.cn/3place/lmyn/LMYN009.html

https://blog.3place.cn/3place/lmyn/LMYN009.html

**Comment by picklecai** [comment]
除了遵守仙人法度之外，
这婆罗门什么都不知道。

坏事一样不懂。真是好孩子、好学生典范。
你告诉我这也是哪家温室里出来的？回答说是森林。可见，即使是编神话故事的人，也深信，人才能让一个人变坏。小修在深山，大修在世上。见过千千万万的坏，还能保持好，才是真正的修炼。

**Comment by picklecai** [comment]
光辉的卢摩婆陀，
威武有力声名扬，
将把鸯伽王来做。
这个国王犯错误，
……

只要做了王，就光辉，就威武有力。总之还是个完人。
全天下百姓都无法活命了，这种惊天动地的惨状，只不过是完人犯的一个错误而已。

**Reply by picklecai** [reply]
对应西游记的凤仙郡。

**Comment by picklecai** [comment]
国王呀！要用一切手段，
把毗槃吒迦的儿子弄到。
这是贤人不在朝的灾难呀。
把哩湿耶舍楞迦弄来以后，
国王呀！要对他恭谨敬礼，
把你那女儿散他嫁给他，
尽上礼节，定要全心全意。

要把道德典范当做吉祥物供起来。
这是早于曹刿阶段的。
如果是曹刿，大概是：减税吧，免税吧。这样老百姓才会为您战斗。

**Comment by picklecai** [comment]
就是在这个时候，
光辉的卢摩婆陀，
威武有力声名扬，
将把鸯伽王来做。 

倒过来再看，才发现卢摩婆陀就是鸯伽王。
